### DISCUSSION TITLE
Themes / Colour Change

### DISCUSSION THREAD

**Discussion by RobHammondIt** [foldynl/QLog#498] [Q&A]
Hi , I've installed Qlog on Kali with GTK3 and flatpak, I'm wondering if it's possible to change the application to Dark theme. I've used both the flatpak overide method for themes and flatseal to set an alternative theme for the application to give me Dark Mode but it constantly just defaults to it's standard look. Before I did this I was trying to set the theme in qt5ct but this was not working either and I assumed that this was because the flatplak install has sandboxed the app to it's settings. Any help with this appreciated. Running light colours over long time periods hurt my eyes.

**Comment by foldynl** [comment]
maybe the QLog wiki will help you

https://github.com/foldynl/QLog/wiki/Main-Window#qt-style-and-dark-mode
